Dockerfile.dev

```diff
@@ -0,0 +1,56 @@
+FROM node:10 as frontend-builder
+
+WORKDIR /frontend
+COPY package.json package-lock.json /frontend/
+RUN npm install
+
+COPY client /frontend/client
+COPY webpack.config.js /frontend/
+RUN npm run build
+
+FROM python:3.7-slim
```

Comment:
@arikfr I just noticed this, were we okay with not using the debian base image anymore? This was done in the Python 3 move and I noted it there but I missed if this was discussed somewhere.

Reply:
The main reason we created `redash/base` image is to trim down the time it takes to build the image. I noticed that it didn't affect it, so kept as is for now. If the need arrises, we can update `redash/base` to Python 3.

Reply:
That makes sense, thanks for the clarification!